command/csi_volume_deregister.go

```diff
@@ -0,0 +1,81 @@
+package command
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/posener/complete"
+)
+
+type CSIVolumeDeregisterCommand struct {
+	Meta
+}
+
+func (c *CSIVolumeDeregisterCommand) Help() string {
+	helpText := `
+Usage: nomad csi volume deregister [options] <input>
+
+  Deregister is used to create or update a CSI volume specification. The volume
```

Comment:
Similar to the above, this should probably read something like `Creates or updates a CSI volume specification.`